src/pages/availability-panel/index.tsx

```diff
@@ -9,14 +9,19 @@ import updateTasksStatus from '@/helperFunctions/updateTasksStatus';
 import { AVAILABLE } from '@/constants/task-status';
 import { FEATURE } from '@/constants/task-type';
 import { BASE_URL } from '@/constants/url';
+import { useGetIdleMembersQuery } from '@/app/services/membersApi';
 
 const AvailabilityPanel: FC = () => {
-    const [idleMembersList, setIdleMembersList] = useState<string[]>([]);
     const [unAssignedTasks, setUnAssignedTasks] = useState<task[]>([]);
     const [error, setError] = useState<boolean>(false);
     const [isTaskLoading, setIsTaskLoading] = useState<boolean>(true);
```

Comment:
Also, see if these states are required or if can we use the error, loading property from the RTK query that you have written or already written RTK query.

Reply:
try to minimize the use state hook if possible

Reply:
These are for a different fetch call.

Reply:
do we need fetch calls if we are using rtk query, do figure it out

Reply:
Ideally no, The Issue this PR solves only asks to migrate idle members fetch call. Would be happy to migrate the other fetch call in a different PR if no one else is working on it.